code/controllers/subsystem/time_track.dm

```diff
@@ -93,7 +93,6 @@ SUBSYSTEM_DEF(time_track)
 		time_dilation_avg_fast = MC_AVERAGE_FAST(time_dilation_avg_fast, time_dilation_current)
 		time_dilation_avg = MC_AVERAGE(time_dilation_avg, time_dilation_avg_fast)
 		time_dilation_avg_slow = MC_AVERAGE_SLOW(time_dilation_avg_slow, time_dilation_avg)
-		GLOB.glide_size_multiplier = (current_byondtime - last_tick_byond_time) / (current_realtime - last_tick_realtime)
```

Comment:
Very interesting. Did you know this line can runtime? I reported this as an issue #75221

Reply:
I uh, don't know how that's possible, but I suppose so yes